homeassistant/components/switch/command_switch.py

```diff
@@ -66,14 +93,24 @@ def is_on(self):
         """ True if device is on. """
         return self._state
 
+    def update(self):
+        """ Update device state. """
+        if self._command_state and self._value_template:
+            payload = CommandSwitch._query_state(self._command_state)
+            payload = template.render_with_possible_json_value(
+                self._hass, self._value_template, payload)
+            self._state = (payload == "True")
```

Comment:
To be consistent with other template implementations, you should convert the payload to lower case:

```
self._state = (payload.lower() == "true")
```